### DISCUSSION TITLE
Is there any free model for the Japanese language ?

### DISCUSSION THREAD

**Discussion by johanburati** [openai/whisper#1266] [General]
I could find the models for the English language but not for the Japanese one, is there any model for the Japanese language freely available ?

**Comment by phineas-pta** [comment]
> find the models for the English language

where did you look up ?

also the default models support many languages, if the results aren't satisfied, do you mean fine-tuned models ? there are plenty on huggingface

**Comment by Tex2002ans** [comment]
The default models handle all languages (including Japanese):

- `tiny`
- `base`
- `small`
- `medium`
- `large`

There are also special `.en` versions—which are fine-tuned for English audio—for every model below `large`:

- `tiny.en`
- `base.en`
- `small.en`
- `medium.en`

(The accuracy is slightly better for English compared to the all-language versions, but it will do worse in non-English audio.)

For more info, see the ["Available models and languages"](https://github.com/openai/whisper#available-models-and-languages) section in the Whisper readme.

**Comment by johanburati** [comment]
Thank you, did not realized that the default models could handle multiple languages.
